### PR TITLE
Request a list of sb_ids associated with a given capture block ID

### PR DESCRIPTION
**Pull request by bngcebetsha**
In this PR, added a method on katportalclient for users to query katportal for sb IDs associated with a given capture block ID. An example is shown on `examples/get_schedule_block_info.py`

*Definition of Done Checklist*

- [x] Code meets our [python style guidelines](https://docs.google.com/document/d/1aZoIyR9tz5rCWr2qJKuMTmKp2IzHlFjrCFrpDDHFypM/edit?usp=sharing)?
- [x] Unit tested (coded, passed, included)?
- [x] I have requested >= 2 reviewers for this PR?
- [x] I have commented my code, particularly in hard-to-understand areas?
- [ ] I have made corresponding changes to the documentation (e.g. Python documentation, System Engineering Documentation, version description updates, README file, etc)?

JIRA: [MT-660](https://skaafrica.atlassian.net/browse/MT-660)

### CODE REVIEW

File: katportalclient/test/test_client.py [hunk 1]
```diff
@@ -590,6 +592,26 @@ def test_schedule_block_detail(self):
...
+            valid_response=r"""
+                {"result":
+                    "[{\"capture_block_id\":\"1556092345\",\"owner\":\"CAM\",\"type\":\"OBSERVATION\",\"sub_nr\":1},
+                      {\"capture_block_id\":\"1556062104\",\"owner\":\"CAM\",\"type\":\"OBSERVATION\",\"sub_nr\":2}
+                     ]"
+                }""",
+            invalid_response="""{"result":null}""",
+            starts_with=capture_block_base_url)
+
+        sb_ids = yield self._portal_client.sb_ids_by_capture_block()
```
Comment (lanceWilliams) [inline katportalclient/test/test_client.py]: It seems like this unittest fails on the number of provided arguments. Does the sb_ids_by_capture_block function work correctly when you provide no arguments?

File: examples/get_schedule_block_info.py [hunk 2]
```diff
@@ -79,6 +79,14 @@ def main():
         #              u'outcome': u'UNKNOWN',
         #              u'data_quality': None}
 
+    cb_ids = ['1555494792.25']
+    # cb_ids = yield portal_client.schedule_blocks_completed()
```
Comment (lanceWilliams) [inline examples/get_schedule_block_info.py]: Was this line here for testing or is it something that got left out?

File: katportalclient/test/test_client.py (hunk 1 above)
Comment (bngcebetsha) [reply katportalclient/test/test_client.py]: You are right, I blindly mimicked a similar test. See latest commit for the fix.

File: examples/get_schedule_block_info.py (hunk 2 above)
Comment (bngcebetsha) [reply examples/get_schedule_block_info.py]: I thought I will need to add a method in portal_client, namely `schedule_blocks_completed`, but at the standup I realised I actually don't need to. Will remove this line now.

File: examples/get_schedule_block_info.py [hunk 3]
```diff
@@ -79,6 +79,13 @@ def main():
         #              u'outcome': u'UNKNOWN',
         #              u'data_quality': None}
 
+    cb_ids = ['1555494792.25']
+    if len(cb_ids) > 0:
+        cb_detail = yield portal_client.sb_ids_by_capture_block(cb_ids[0])
```
Comment (ajoubertza) [inline examples/get_schedule_block_info.py]: All the changes to this example script were just used for testing.  They do not relate to the script's original purpose of "getting schedule block info".  Please remove them.  This PR should not be changing this example.

If you think an example of getting SBs linked to a  CB is necessary, then a new example script dedicated to that should be created.

File: katportalclient/client.py [hunk 4]
```diff
@@ -1105,6 +1112,44 @@ def schedule_block_detail(self, id_code):
                 "Invalid schedule block ID: " + id_code)
         raise tornado.gen.Return(schedule_block)
 
+    #TODO: Might need to add a usage script in ../examples
```
Comment (ajoubertza) [inline katportalclient/client.py]: This TODO can be removed - not really something belongs here.

File: katportalclient/client.py [hunk 5]
```diff
@@ -1105,6 +1112,44 @@ def schedule_block_detail(self, id_code):
...
+
+        Parameters
+        ----------
+        capture_block_id: str
+            Capture block identifier. For example: ``1556067480``.
+
+        Returns
+        -------
+        list:
+            List of scheduled block ID strings.  Ordered according to
```
Comment (ajoubertza) [inline katportalclient/client.py]: Typo:  `scheduled` -> `schedule`

File: katportalclient/client.py [hunk 6]
```diff
@@ -1105,6 +1112,44 @@ def schedule_block_detail(self, id_code):
...
+            priority of the schedule blocks (first has hightest priority).
```
Comment (ajoubertza) [inline katportalclient/client.py]: Also they are not ordered by priority, so remove (that comes from the copied docstring)

File: katportalclient/test/test_client.py [hunk 7]
```diff
@@ -195,6 +195,7 @@ def mock_fetch(url):
                        {'websocket': self.websocket_url,
                         'historic_sensor_values': r"http://0.0.0.0/history",
                         'schedule_blocks': r"http://0.0.0.0/sb",
+                        'capture_blocks': r"http://0.0.0.0/cb/sb/",
```
Comment (ajoubertza) [inline katportalclient/test/test_client.py]: Drop the trailing `/sb/`

File: katportalclient/client.py [hunk 8]
```diff
@@ -1105,6 +1112,44 @@ def schedule_block_detail(self, id_code):
                 "Invalid schedule block ID: " + id_code)
         raise tornado.gen.Return(schedule_block)
 
+    #TODO: Might need to add a usage script in ../examples
+    @tornado.gen.coroutine
+    def sb_ids_by_capture_block(self, capture_block_id):
+        """Return list of approved observation schedule blocks.
```
Comment (ajoubertza) [inline katportalclient/client.py]: The docstring needs to be updated.  Summary line and the paragraph below.

**Comment by ajoubertza** [comment]
And please get the tests on Jenkins to pass.

**Pull request merged**